app/views/mentee_mailer/welcome.html.erb

```diff
@@ -8,12 +8,12 @@ A little bit about myself:<br />
 </p>
 
 <p>
-The best way to start this process is for me to learn more about you. Getting an understanding of your military background and your programming goals,
+The best way to start this process is for me to learn more about you.Getting an understanding of your military background and your programming goals,
 whether they be pursuing a new career, furthering an existing career, or simply learning to code as a hobby, will be helpful in matching you up with the
-right mentoring groups. It’d be great for us to connect on an initial call so I can go over our “checklist” with you. Would you be available for a quick chat?
+right mentoring groups.It’d be great for us to connect on an initial call so I can go over our “checklist” with you. Would you be available for a quick chat?
 </p>
 
 <p>
-Look forward to hearing from you!<br />
+Looking forward to hear from you!<br />
```

Comment:
This change doesn't make much sense grammatically. Maybe `Looking forward to hearing from you` instead?